### PR TITLE
add venue support for telegram alarm

### PR DESCRIPTION
**Pull request by AstAst42**
## Description
Adds possibility to use venues in telegram alarms. Venues are very similar to maps, but they can have additional text just below the text. This makes it easier to understand which "map" (venue) message belongs to which mon text message and makes it easier to share the alarm by simply sharing the venue only. You can edit the venue text like all other texts but keep in mind that the space is limited.

image

This text is also shown after pressing on the venue if you want to use navigation.

image

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context

This makes it easier to find the Mons you are looking for because you can combine mon values with a map and improves sharing.

## How Has This Been Tested?

I used this as a local change on a server for 3 small cities, had it run for months and had no problems.

## Wiki Update

- [x] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

### CODE REVIEW

**Comment by LegitDongo** [comment]
This seems like it would cause unexpected results for those who use current message functionality with the venue option on

Can someone confirm that this works as expected even without the venue message option?

File: PokeAlarm/Alarms/Telegram/TelegramAlarm.py [hunk 1]
```diff
@@ -24,54 +24,63 @@ class TelegramAlarm(Alarm):
...
+                  'sticker_notify', 'message', 'venue_message',
+                  'message_notify', 'venue',
                   'venue_notify', 'map', 'map_notify', 'max_attempts',
                   'web_preview'])
 
     _defaults = {  # No touchy!!! Edit alarms.json!
         'monsters': {
             'message': "*A wild <mon_name> has appeared!*\n"
                        "Available until <24h_time> (<time_left>).",
+            'venue_message': "",
```
Comment (billyjbryant) [inline PokeAlarm/Alarms/Telegram/TelegramAlarm.py:36]: Wouldn't the proper thing here be to set `venue_message` to `null` rather than an empty string?

**Comment by billyjbryant** [comment]
Is it possible to disable this in settings so that Venues show only if they are either enabled or not disabled?

**Review by billyjbryant — CHANGES_REQUESTED** [review]
Please address the comments on this PR

**Comment by billyjbryant** [comment]
This PR appears to be abandoned, please re-open and address comments if you wish to introduce this feature.

**Pull request closed without merging**